messages_example_test.go

```diff
@@ -0,0 +1,89 @@
+// +build go1.10
+
+package mssql_test
+
+import (
+	"context"
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	mssql "github.com/denisenkom/go-mssqldb"
+	"github.com/golang-sql/sqlexp"
+)
+
+const (
+	msgQuery = `select name from sys.tables
```

Comment:
I think it would be better to select something that won't vary from system to system (and potentially even run to run). That would make the test more repeatable.